presidio-analyzer/presidio_analyzer/recognizer_registry/recognizer_registry_provider.py

```diff
@@ -281,17 +291,26 @@ def create_recognizer_registry(self) -> RecognizerRegistry:
 
         for recognizer_conf in recognizer_instances:
             if isinstance(recognizer_conf, PatternRecognizer):
-                recognizer_conf.global_regex_flags = fields["global_regex_flags"]
+                recognizer_conf.global_regex_flags = self.global_regex_flags
 
-        recognizer_instances = [
-            recognizer
-            for recognizer in recognizer_instances
-            if recognizer.supported_language in self.supported_languages
-        ]
+        index = 0
```

Comment:
Feels like this can get bad easily in the future, can be simplified with less mutability, maybe?
`filtered_recognizer_instances = [r for r in recognizer_instances if language_supported_globally(r)]`
and print inside language_supported_globally
Or with `filterfalse

Reply:
Love it, deal